phobos/ci/model_testing.py

```diff
@@ -25,15 +25,15 @@ def __init__(self, new_model, compare_model=None):
         self.old_hyrodyn = None
         self.new_hml_test = get_load_report(self.new.robot.xmlfile, self.new.robot.submechanisms_file)
         self.new_sm_hml_test = []
-        if len([x for x in self.new.robot.submodel_defs if not x["name"].startswith("#submech#")]) > 0:
+        if len([x for x in self.new.robot.submodel_defs if not x.startswith("#submech#")]) > 0:
```

Comment:
If submodel_defs is a dict for you (as it is for me) why are you doing this change?